apps/settings/elements/firefox_accounts.html

```diff
@@ -29,14 +29,14 @@ <h2 data-l10n-id="fxa-confirm-account">Confirm Your Account</h2>
       </header>
       <ul>
         <li class="description">
-          <p data-l10n-id="fxa-verification-email-sent-msg" id="fxa-unverified-text">
+          <p id="fxa-unverified-text">
             We will send an email to: foo@bar.com
           </p>
           <p data-l10n-id="fxa-please-verify">
             Click the verification link in your email to confirm your account.
           </p>
-          <p id="fxa-resend-email">
-            Don't see an email? <a href="#">Resend</a>.
+          <p id="fxa-resend-email" data-l10n-id="fxa-dont-see-email-resend">
+            Don't see an email? <a href="#" id="fxa-resend">Resend</a>.
```

Comment:
That's what we do in other places, just sth like this:

```
foo=Hello world <a>Click me<a>
```

```
<p data-l10n-id="foo">
  <a href="http://www.mozilla.org" class="highlight"></a>
</p>
```

If you prefer to leave it, I'm ok with that as well. It doesn't affect the result. We're just trying to remove en-US content from HTML files to remove the confusion that happens when we update strings and the content in HTML gets out of sync